### PR TITLE
[docs] 2.4.3.0 release notes

### PR DESCRIPTION
**Pull request by polarweasel**
build preview: https://deploy-preview-8420--infallible-bardeen-164bc9.netlify.app/latest/releases/whats-new/stable-release/

### CODE REVIEW

File: docs/content/latest/releases/whats-new/stable-release.md [hunk 1]
```diff
@@ -21,6 +21,62 @@ Included here are the release notes for all releases in the v2.4 stable release
...
+* [[8348](https://github.com/yugabyte/yugabyte-db/issues/8348)] Correctly handling a failure to create a priority thread pool worker thread
+
+#### Yugabyte Platform
+
+* [[8220](https://github.com/yugabyte/yugabyte-db/issues/8220)] Impersonate user from admin console
+
+### DISCARD THESE?
+
+* [[8176](https://github.com/yugabyte/yugabyte-db/issues/8176)]Use a separate config to override settings in 20-nproc.conf
+* [[8150](https://github.com/yugabyte/yugabyte-db/issues/8150)] [[8196](https://github.com/yugabyte/yugabyte-db/issues/8196)] Fix preceding op id in case of empty ops sent to the follower
```
Comment (bmatican) [inline docs/content/latest/releases/whats-new/stable-release.md:76]: this is a very serious database bugfix, I'd definitely include this, as it's one of the reasons we're trying to get a 2.4.3 out

File: docs/content/latest/releases/whats-new/stable-release.md [hunk 2]
```diff
@@ -21,6 +21,63 @@ Included here are the release notes for all releases in the v2.4 stable release
...
+
+#### Core database
+
+* [[8052](https://github.com/yugabyte/yugabyte-db/issues/8052)] Add ability to configure cipher list
+
+#### Yugabyte Platform
+
+* [[1342](https://github.com/yugabyte/yugabyte-db/issues/1342)] Fixing the error message when the get host info call to gcp fails
+
+* [[6758](https://github.com/yugabyte/yugabyte-db/issues/6758)] Unable to reuse On-premise instance incase previouse universe failed / deleted (#8263)
```
Comment (chirag-yb) [inline docs/content/latest/releases/whats-new/stable-release.md:57]: This is a bug

File: docs/content/latest/releases/whats-new/stable-release.md [hunk 3]
```diff
@@ -21,6 +21,63 @@ Included here are the release notes for all releases in the v2.4 stable release
...
+
+* [[8165](https://github.com/yugabyte/yugabyte-db/issues/8165)]Yugaware creates zombie ssh process
+
+* [[8203](https://github.com/yugabyte/yugabyte-db/issues/8203)] Remove state from cloud initializers to avoid concurrency issues
```
Comment (chirag-yb) [inline docs/content/latest/releases/whats-new/stable-release.md:61]: Its an internal improvement. No need to document this.

File: docs/content/latest/releases/whats-new/stable-release.md [hunk 4]
```diff
@@ -21,6 +21,63 @@ Included here are the release notes for all releases in the v2.4 stable release
...
+
+#### Core database
+
+* [[8150](https://github.com/yugabyte/yugabyte-db/issues/8150)] [[8196](https://github.com/yugabyte/yugabyte-db/issues/8196)] Fix preceding op id in case of empty ops sent to the follower
+
+* [[8348](https://github.com/yugabyte/yugabyte-db/issues/8348)] Correctly handling a failure to create a priority thread pool worker thread
+
+#### Yugabyte Platform
+
+* [[8220](https://github.com/yugabyte/yugabyte-db/issues/8220)] Impersonate user from admin console
```
Comment (chirag-yb) [inline docs/content/latest/releases/whats-new/stable-release.md:73]: Please remove this item

File: docs/content/latest/releases/whats-new/stable-release.md [hunk 5]
```diff
@@ -21,6 +21,63 @@ Included here are the release notes for all releases in the v2.4 stable release
...
+
+### DISCARD THESE?
+
+* [[8176](https://github.com/yugabyte/yugabyte-db/issues/8176)]Use a separate config to override settings in 20-nproc.conf
```
Comment (chirag-yb) [inline docs/content/latest/releases/whats-new/stable-release.md:77]: Its a bug. Should reword it "GCP universes are not having correct nproc system settings"

File: docs/content/latest/releases/whats-new/stable-release.md [hunk 6]
```diff
@@ -21,6 +21,63 @@ Included here are the release notes for all releases in the v2.4 stable release
...
+* [[8173](https://github.com/yugabyte/yugabyte-db/issues/8173)] Remove the third-party submodule and add a text file with thirdparty SHA1
+* [[8041](https://github.com/yugabyte/yugabyte-db/issues/8041)] Preserve environment variables to ansible call
```
Comment (chirag-yb) [inline docs/content/latest/releases/whats-new/stable-release.md:79]: Please remove this line item

**Review by chirag-yb — COMMENTED** [review]
My comments are in

**Review by rao-vasireddy — APPROVED** [review]

**Pull request merged**